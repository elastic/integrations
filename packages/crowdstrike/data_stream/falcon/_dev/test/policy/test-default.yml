vars:
  url: http://host.tld
  client_id: test_client_id
  client_secret: test_client_secret
  token_url: http://host.tld/oauth2/token
data_stream:
  vars:
    preserve_original_event: true
    preserve_duplicate_custom_fields: true
