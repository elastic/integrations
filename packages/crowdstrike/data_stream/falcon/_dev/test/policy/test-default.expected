inputs:
    - data_stream:
        namespace: ep
      meta:
        package:
            name: crowdstrike
      name: test-default-crowdstrike
      streams:
        - allow_deprecated_use: true
          data_stream:
            dataset: crowdstrike.falcon
            type: logs
          exclude_files:
            - \.gz$
          multiline.match: after
          multiline.max_lines: 5000
          multiline.negate: true
          multiline.pattern: ^{
          multiline.timeout: 10
          paths:
            - /var/log/crowdstrike/falconhoseclient/output*
          publisher_pipeline.disable_host: true
          tags:
            - preserve_original_event
            - forwarded
            - crowdstrike-falcon
      type: logfile
      use_output: default
output_permissions:
    default:
        _elastic_agent_checks:
            cluster:
                - monitor
        _elastic_agent_monitoring:
            indices: []
        uuid-for-permissions-on-related-indices:
            indices:
                - names:
                    - logs-crowdstrike.falcon-ep
                  privileges:
                    - auto_configure
                    - create_doc
secret_references: []
